extensions/arc/notebooks/arc/content/postgres/tsg100-troubleshoot-postgres.ipynb

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
Looks like some of these are collapsed by default. Just want to make sure that the cells that are collapsed by default are the cells that are intended (or others that should be collapsed aren't currently marked as such with the hide_input tag)

Reply:
The correct cells are collapsed by default via hide_input.  The pre-requisite/common functions at the beginning are collapsed so that the real work starts with the first open cell.